packages/slice-machine/lib/env/manifest.ts

```diff
@@ -23,6 +25,7 @@ export enum ManifestState {
   MissingEndpoint = "MissingEndpoint",
   InvalidEndpoint = "InvalidEndpoint",
   InvalidJson = "InvalidJson",
+  InvalidFramework = "InvalidFramework",
 }
 
 const Messages = {
```

Comment:
I think this can be cleaned as most of those issues will be handled in the validator